pages/agent/v3/elastic_ci_aws.md.erb

```diff
@@ -43,7 +46,8 @@ Also keep in mind the EC2 HTTP metadata server is available from within builds,
 
 The Buildkite Elastic CI Stack supports:
 
-- All major AWS regions
+- All AWS regions except China and US GovCloud
+- Linux or Windows operating systems
```

Comment:
I debated whether `and` or `or` was appropriate here, a given stack supports one _or_ the other not both. But the stack as a whole can be deployed for each operating system separately. I’ll leave this one to you!

Reply:
Ahah. As this is a broad feature set, I'd lean towards `and`